packages/reactivity/src/ref.ts

```diff
@@ -60,7 +60,7 @@ export function triggerRefValue(ref: RefBase<any>, newVal?: any) {
   }
 }
 
-export type ToRef<T> = [T] extends [Ref] ? T : Ref<UnwrapRef<T>>
+type ToRef<T> = [T] extends [Ref] ? T : Ref<UnwrapRef<T>>
```

Comment:
This should be moved to above `toRef` and exported in `index.ts` since it's used in the return type of a public API.

Reply:
Sorry, don't quite understand. Is this all right?